zerver/lib/send_email.py

```diff
@@ -137,13 +141,16 @@ def render_templates() -> Tuple[str, str, str]:
         )
 
         try:
-            html_message = loader.render_to_string(template_prefix + ".html", context)
+            html_message = loader.render_to_string(template_prefix + ".source.html", context)
         except TemplateDoesNotExist:
-            emails_dir = os.path.dirname(template_prefix)
-            template = os.path.basename(template_prefix)
-            compiled_template_prefix = os.path.join(emails_dir, "compiled", template)
-            html_message = loader.render_to_string(compiled_template_prefix + ".html", context)
-        return (html_message, message, email_subject)
+            # What should we do here since we don't have compiled htmls anymore?
```

Comment:
What should we do here since we don't have compiled htmls anymore?

Reply:
So the previous logic was just hackily look for the template in two paths -- the `compiled/` directory and the main one. Since we don't need to compile emails anymore, I suspect you can just get rid of all the complexity -- replace `.source.html` with just `.html` in the filenames, just look at the original path, and not catch the exception -- if the email template doesn't exist, it's a bug.